doc/api/cli.md

```diff
@@ -51,6 +51,37 @@ entry point, the `node` command will accept as input only files with `.js`,
 [`--experimental-wasm-modules`][] is enabled; and with no extension when
 [`--experimental-default-type=module`][] is passed.
 
+## Sub-commands
+
+### `node run <command> [<arguments>]`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+This runs an arbitrary command from a package.json's `"scripts"` object.
+If no `"command"` is provided, it will list the available scripts.
+
+By default, this command adds the current path appended by
+`node_modules/.bin` directory to the PATH in order to execute the binaries
+from dependencies.
+
+For example, the following command will run the `test` script of
+the current project's `package.json`:
+
+```console
+$ node run test
+```
+
+You can also pass arguments to the script, any argument after the script
+name will be appended to the script:
+
+```console
+$ node run test --verbose
+```
+
```

Comment:
This needs an extra section explaining differences from `npm run`, including which of those differences we might address in the future and which we intend to keep as differences. I agree with the others who commented that we don’t want this feature expanding indefinitely in scope, and we should define its intended DX up front in order to prevent people from opening lots of issues for it when it doesn’t behave as _they_ expect. We can say that performance is a top priority for this feature, and that’s a big reason why we aren’t implementing many of the features that `npm run` offers such as folder searching for `package.json` files or running pre/post scripts. The intent behind `node run` is to provide a _simple_ and _fast_ way to run `package.json` scripts, not to move `npm run` into core; if users want the full DX of `npm run`, they should just use that.

Reply:
We can add this after landing this pull request. I'm not in favor of comparing a Node.js functionality with something else on Node.js documentation, but if that's the case, please let's leave it for another PR discussion.

Reply:
```suggestion

#### Intentional limitations

`node run` is not meant to match the behaviors of `npm run` or of the `run`
commands of other package managers. The Node.js implementation is intentionally
more limited, in order to focus on top performance for the most common use
cases.

Some features of other `run` implementations that are intentionally excluded
are:
* Searching for `package.json` files outside of the current folder.
* Prepending the `.bin` or `node_modules/.bin` paths of folders outside of the
  current folder.
* Running `pre` or `post` scripts in addition to the specified script.
* Defining package manager-specific environment variables.
* Setting `process.title`.
```

Reply:
Added. Thank you for helping me.